DirectEmployers/MyJobs
# Pd 3579 traffic reports datasource

## What's this PR do?
Created 3 datasources for one for each of the newly created analytics tables
updated schema to include site-traffic with newly created categories
## Screenshots (if appropriate)

## Where should the reviewer start?
Check for Site-traffic in analytics report
## How should this be manually tested?
Add a new waffle switch `new-site-traffic`
Ensure the 3 newly created tables have some data to test with.
Click-on Site traffic reports and drill down through various categories
## Any background context you want to provide?
This PR has combos put in and currently 1 button at a time is clickable.
The drilldown on combo selection is not as expected
